app/helpers/dashboard_helper.rb

```diff
@@ -13,11 +36,41 @@ def projects_dashboard_filter_path(options={})
     path
   end
 
-  def assigned_issues_dashboard_path
-    issues_dashboard_path(assignee_id: current_user.id)
+  def assigned_entities_count(current_user, entity, scope = nil)
+    items = current_user.send("assigned_" + entity.pluralize).opened
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.